### PR TITLE
update to activiti-cloud-application repository

### PR DESCRIPTION
**Pull request by robfrank**

### CODE REVIEW

File: fetch-versions.sh [hunk 1]
```diff
@@ -70,11 +70,11 @@ for project in "${projects[@]}"; do
...
   'activiti-cloud')
-    file=repos-activiti-cloud-mono.txt
-    ;;
-  'activiti-cloud-dependencies')
     file=repos-activiti-cloud.txt
-    examples_file=repos-activiti-cloud-examples.txt
+    ;;
+  'activiti-cloud-application')
+    file=repos-activiti-cloud-application.txt
+#    examples_file=repos-activiti-cloud-examples.txt
```
Comment (miguelruizdev) [inline fetch-versions.sh:77]: any reason why we leave it commented out?

Comment (robfrank) [reply fetch-versions.sh:77]: good question, I'm reviewing

Comment (robfrank) [reply fetch-versions.sh]: done

Comment (miguelruizdev) [reply fetch-versions.sh]: good 👍

**Pull request merged**